devdocs/settingUp.md

```diff
@@ -25,7 +25,7 @@ Important: When a version is specified, please install that version instead of t
 ##Setting up the dev server
 `Dev server` means running the server in your own computer.
 
-1. Fork our repo at `https://github.com/TEAMMATES/repo. Clone that fork to your hard disk.
+1. Fork our repo at https://github.com/TEAMMATES/repo. Clone that fork to your hard disk.
```

Comment:
Please try not to fix things unrelated to the issue that you're working on - it makes it harder if we ever need to revert a PR - could you revert this commit and check if an issue is already opened regarding this and if not please open one instead